definitions/checks/sca_only.rb

```diff
@@ -0,0 +1,42 @@
+class Checks::CheckOrganizationContentAccessMode < ForemanMaintain::Check
+  metadata do
+    label :check_organization_content_access_mode
+    description 'Check if any organizations are using EntitlementContentAccess mode before upgrading'
+    tags :pre_upgrade
+
+    confine do
+      feature(:katello)
+    end
+  end
+
+  def run
+    with_spinner('Checking organization content access modes') do
+      output = execute_rake_task
+      orgs_to_migrate = parse_orgs_to_migrate(output)
+
+      assert(
+        orgs_to_migrate.empty?,
+        "The following organizations are using EntitlementContentAccess mode:\n"\
```

Comment:
"EntitlementContentAccess" is a term I hadn't heard before. I prefer just "entitlement mode."

```suggestion
        "The following organizations are using entitlement mode:\n"\
```

Reply:
conveniently, this should get a few lines under rubocop's length limit as well